docs/changelog/3_x.rst

```diff
@@ -0,0 +1,729 @@
+====
+v3.0
+====
+
+:edb-alt-title: EdgeDB v3
+
+The EdgeDB 3.0 beta was released on May 5th, 2023. We would like to thank our
+community for reporting issues and contributing fixes. You are awesome! ❤️
+
+To play with the new features, install the CLI using
+<https://www.edgedb.com/install>`_, upgrade to the testing channel using
+``edgedb cli upgrade --to-testing``, and initialize a new project.
+
+.. code-block:: bash
+
+  $ edgedb project init --server-version=testing
+
+
+Upgrading
+=========
+
+**Local instances**
+
+To upgrade a local project, first upgrade your CLI using ``edgedb cli upgrade
+--to-testing``. Then run the following command inside the project directory.
+
+.. code-block:: bash
+
+  $ edgedb project upgrade --to-testing
+
+Alternatively, specify an instance name if you aren't using a project.
+
+.. code-block:: bash
+
+  $ edgedb project upgrade --to-testing -I my_instance
+
+**Hosted instances**
+
+To upgrade a remote (hosted) instance, we recommend the following
+dump-and-restore process.
+
+1. Spin up an empty 3.0 instance. You can use one of our :ref:`deployment
+   guides <ref_guide_deployment>`, but you will need to modify some of the
+   commands to use our testing channel and the beta release.
+
+   Under Debian/Ubuntu, when adding the EdgeDB package repository, use this
+   command instead:
+
+   .. code-block:: bash
+
+       $ echo deb [signed-by=/usr/local/share/keyrings/edgedb-keyring.gpg] \
+           https://packages.edgedb.com/apt \
+           $(grep "VERSION_CODENAME=" /etc/os-release | cut -d= -f2) testing \
+           | sudo tee /etc/apt/sources.list.d/edgedb.list
+
+   Use this command for installation under Debian/Ubuntu:
+
+   .. code-block:: bash
+
+       $ sudo apt-get update && sudo apt-get install edgedb-3-beta1
+
+   Under CentOS/RHEL, use this installation command:
+
+   .. code-block:: bash
+
+       $ sudo yum install edgedb-3-beta1
+
+   In any required ``systemctl`` commands, replace ``edgedb-server-2`` with
+   ``edgedb-server-3``.
+
+   Under any Docker setups, supply the ``3.0-beta.1`` tag.
+
+2. Take your application offline, then dump your v2.x database with the CLI
+
+   .. code-block:: bash
+
+       $ edgedb dump --dsn <old dsn> --all my_database.dump/
```

Comment:
do we need `--format=dir` here? this is in the help output:

```
OPTIONS:
        --all                Dump all databases and the server configuration. path is a
                             directory in this case
        --format <format>    Choose dump format. For normal dumps this parameter should be
                             omitted. For --all only --format=dir is required.
                              [possible values: dir]
```

Reply:
Good catch!